docs/api/saved-objects/find.asciidoc

```diff
@@ -0,0 +1,60 @@
+[[saved-objects-api-find]]
+=== Find Objects
+
+experimental[This functionality is *experimental* and may be changed or removed completely in a future release.]
+
+The find saved object API enables you to retrieve a paginated set of Kibana
+saved objects by various conditions.
+
+==== Request
+
+`GET /api/saved_objects/_find`
+
+==== Query Parameters
+
+`per_page` (optional)::
+  (number) The number of objects to return per page
+`page` (optional)::
+  (number) The page of objects to return
+`type` (optional)::
+  (array|string) The saved object type or types that the response should be limited to
+`search` (optional)::
+  (string) A {ref}/query-dsl-simple-query-string-query.html[simple_query_string] Elasticsearch query to filter the objects in the response
+`search_fields` (optional)::
+  (string) The fields to perform the `simple_query_string` parsed query against
+`sort_field` (optional)::
+  (string) The field on which the response will be sorted
+`fields` (optional)::
+  (string) The fields to return in the response
+
+
+==== Examples
+
+The following example attempts to find index patterns with titles that start
+with `my`:
+
+[source,js]
```

Comment:
I've pushed a change that clarifies this behavior, but I think it really belongs in some top level "API conventions" documentation since its not unique to the saved objects API. But we can address that separately.